04/app.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
 import File from './File';
 import List from './List';
+// import data from './static/article.txt'
 
 class App extends React.Component {
     state = {
         filesList: [],
     }
-
+    handleFilesSubmit(newObject) {
```

Comment:
Gdybyś zrobił inicjalizator właściwości tj. `handleFilesSubmit = newObject =>{ /* ... */ }` to byś nie musiał wykorzystywać `.bind()`